lib/std/math/modf.zig

```diff
@@ -133,8 +133,6 @@ test "math.modf" {
     // NOTE: No struct comparison on generic return type function? non-named, makes sense, but still.
     try expect(a.ipart == b.ipart and a.fpart == b.fpart);
 
-    const c = modf(@as(f64, 1.0));
-    const d = modf64(1.0);
     try expect(a.ipart == b.ipart and a.fpart == b.fpart);
```

Comment:
```suggestion
    try expectEqual(c, d);
```
If you're feeling fancy replace the `expect` above with an `expectEqual`.